### ISSUE TITLE
Content is not Syncing with Youtube accounts

### ISSUE THREAD

**Issue by anujbalak** [kotlindevs/Dopamine#2]
I think, signin has nothing to do with the app because it does not fetch watch history, playlist and the video suggetions. Either remove signin or make worth to signin in the app.

**Comment by kotlindevs** [comment]
Yep I know but including a login module is just a demo for project .
